custom/staking/src/lib.rs

```diff
@@ -1314,6 +1346,62 @@ decl_module! {
             Self::deposit_event(RawEvent::InvalidatedNominators(caller, expired_nominators));
         }
 
+        /// Enables individual commisions. This can be set only once. Once individual commission
+        /// rates are enabled, there's no going back.  Only Governance committee is allowed to
+        /// change this value.
+        #[weight = SimpleDispatchInfo::FixedOperational(100_000)]
+        fn enable_individual_commissions(origin) {
+            T::RequiredCommissionOrigin::try_origin(origin)
```

Comment:
Does this need to default the individual commissions to the previous global commission?